specification/sdk-tracing.md

```diff
@@ -260,6 +260,40 @@ high contention in a very high traffic service.
 * `maxExportBatchSize` - the maximum batch size of every export. It must be
   smaller or equal to `maxQueueSize`. The default value is `512`.
 
+##### Span watcher processor
+
+This implementation passes spans that have been started but not yet ended
+to the configured exporter.
+On each interval defined by `reportIntervalMillis`, all unfinished spans that
+have been started more than `reportIntervalMillis` ago, are exported as is.
```

Comment:
This implies that one more, parallel export is started if the previous has not yet finished. This may result in a lot parallel exports on hanging connections/bad set intervals.

Reply:
Should we define the report interval as the duration between two exports (export1.end -> interval -> export2.start), i.e., define the export as blocking rather than having a timer that triggers a new export every n millis?
An alternative would be to specify that exports should be skipped if a previous export is still running at the next timer interval.